packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -124,10 +141,18 @@ final debugPaintBaselines = ToggleableServiceExtensionDescription<bool>._(
   icon: createImageIcon('icons/inspector/textArea@2x.png'),
   enabledValue: true,
   disabledValue: false,
-  enabledTooltip: 'Hide Paint Baselines',
-  disabledTooltip: 'Show Paint Baselines',
   gaScreenName: analytics_constants.inspector,
   gaItem: analytics_constants.paintBaseline,
+  tooltipBuilder: (bool isSelected, Widget child) {
+    return RichTooltip(
+      message:
+          'Show baselines, which are used for aligning text. Can be useful for checking if text is aligned.',
```

Comment:
For consistency make these tooltips describe the action that will be taken on click as they did before. E.g.
`'${isSelected ? 'Show' : 'Hide'} baselines, which are...'`
I

Reply:
or change all the tootips to just have a single description of what the button does as the current approach may be more confusing than helpful.

Reply:
Will do; my preference is having the same tooltip text - I think it's less confusing.

Reply:
Sounds good. Lets remove all the tooltips that change depending on state. I think we did that because the button states were hard to track.